Examples/Algorithms/Geant4/src/SimParticleTranslation.cpp

```diff
@@ -76,6 +76,7 @@ void ActsExamples::SimParticleTranslation::GeneratePrimaries(G4Event* anEvent) {
   std::optional<Acts::Vector4> lastVertex;
 
   constexpr double convertLength = CLHEP::mm / Acts::UnitConstants::mm;
+  constexpr double convertTime = CLHEP::ns / Acts::UnitConstants::ns;
```

Comment:
Should we centralize these definitions at some point?

Reply:
that would be good I think - but I would rather do that in a separate PR